app/static/src/app/components/sessions/SessionsPage.vue

```diff
@@ -2,15 +2,66 @@
   <div class="container">
     <div class="row">
       <h2>Sessions</h2>
-      <div> Coming soon - Sessions list!</div>
+    </div>
+    <template v-if="sessionsMetadata">
```

Comment:
There's some logic here which I don't really understand but is pretty cool. If you navigate to the sessions page and return with back it does seem to load the correct session (the url ends up as http://localhost:3000/apps/day1/?sessionId=a8c2377d48e713bb5fb31ec630538452) which was a surprise - navigating directly to that URL triggers the fake rehydration so it looks like we actually have sharing even if it's a bit nasty)

It would be good if this page had some navigational element that gave the user a hint as to how to return to their work (cancel, back, an X?)

Reply:
It's nice how navigating to apps/day1/sessions loads the sessions page too

Reply:
Actually, that's a good point - when you navigate back from the session list for a loaded session it shouldn't reload with the session id query param as that will retrigger rehydrate which you don't really want as it will overwrite any subsequent changes you've made in the current session with what was in the session you first loaded. I'll look into sorting that out in the router.

Reply:
Or maybe we get away with it because it doesn't contact the backend when it goes through the vue router - I'll check!

Reply:
Yeah, it's fine actually, it doesn't trigger a re-rehydrate when you navigate back from the Sessions page, though the url becomes somewhat misleading once you start making changes in your loaded copy of the session I guess.